tools/build.py

```diff
@@ -249,7 +249,8 @@
                                                         clean=options.clean,
                                                         archive=(not options.no_archive),
                                                         macros=options.macros,
-                                                        name=options.artifact_name)
+                                                          name=options.artifact_name,
```

Comment:
My auto-indenter fights this code base. Sometimes it wins. I'll rebase this out.